src/structures.rs

```diff
@@ -224,7 +219,7 @@ impl fmt::Display for SvParameter {
                 writeln!(f, "    DataType: {:?}", x)?;
             }
         }
-        writeln!(f, "    DataTypeStatus: {:?}", self.datatype_status)?;
+        writeln!(f, "    DataTypeStatus: {:?}", self.datatype_overridable)?;
```

Comment:
DataTypeStatus -> DataTypeOverridable

Reply:
Fixed.

Reply:
> Is this still a draft, or ready for merge?

It is still a draft, I'm trying to fix the bug found in the other PR and I still have a lot of work to do in this PR.